hal/armv7a/zynq7000/timer.c

```diff
@@ -72,13 +60,38 @@ static int _timer_irqHandler(unsigned int n, cpu_context_t *ctx, void *arg)
 }
 
 
-void hal_setWakeup(u32 when)
+void hal_timerSetWakeup(u32 when)
 {
 
 }
 
 
-time_t hal_getTimer(void)
+time_t hal_timerUs2Cyc(time_t us)
+{
+	return (55555LL * us) / 1000LL;
+}
+
+
+time_t hal_timerCyc2Us(time_t cyc)
+{
+	return (cyc * 1000LL) / 55555LL;
+}
+
+
+time_t hal_timerGetUs(void)
```

Comment:
Why is this exact copy of `hal_timerGetCyc`? Looking at functions above `cyc != us` on this arch.

Reply:
Functions are not the same.

Reply:
Right, not exact. But still, this function can mostly replaced by call to `hal_timerGetCyc`

Reply:
Done